bin/steps/conda_compile

```diff
@@ -13,6 +13,11 @@ if [ ! -d /app/.heroku/miniconda ]; then
     bash ${MINICONDA_VERSION}-Linux-x86_64.sh  -p /app/.heroku/miniconda/ -b | indent
     rm -fr ${MINICONDA_VERSION}-Linux-x86_64.sh
 
+    if [ -f .condarc ]; then
+	    puts-step "Copying condarc file in home directory"
```

Comment:
🐄 "Copying condarc file _to_ home directory"

Also if you decide to fixup this commit, there are a couple of typos in the commit message ("wich" and "fiferent").

Reply:
OK will fix these, thank you! sorry for the typos.

Reply:
(FYI I'm also going to change that to _miniconda directory_ instead of _home directory_ as I changed that yesterday because of permission issues, and `.condarc` can be in any of those dirs )